<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shipment_00</class>
 <widget class="QWizardPage" name="Shipment_00">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>489</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string notr="true">Medication</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="medicationLabel">
          <property name="toolTip">
           <string comment="MEDICATION DISPLAY TOOLTIP">MEDICATION</string>
          </property>
          <property name="text">
           <string notr="true">PH MEDICATION</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="strengthLabel">
          <property name="toolTip">
           <string comment="STRENGTH DISPLAY TOOLTIP">STRENGTH</string>
          </property>
          <property name="text">
           <string notr="true">PH STRENGTH</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="formLabel">
          <property name="toolTip">
           <string comment="FORM DISPLAY TOOLTIP">FORM</string>
          </property>
          <property name="text">
           <string notr="true">PH FORM</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="unitSizeLabel">
          <property name="toolTip">
           <string comment="UNIT SIZE DISPLAY TOOLTIP">UNIT SIZE</string>
          </property>
          <property name="text">
           <string notr="true">PH UNIT SIZE</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="expireLabel">
       <property name="toolTip">
        <string comment="EXPIRATION EDIT TOOLTIP">EXPIRATION</string>
       </property>
       <property name="text">
        <string comment="EXPIRATION EDIT LABEL">EXPIRATION</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDateEdit" name="expireField">
         <property name="toolTip">
          <string comment="EXPIRATION EDIT TOOLTIP">EXPIRATION</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lotLabel">
       <property name="toolTip">
        <string comment="LOT EDIT TOOLTIP">LOT</string>
       </property>
       <property name="text">
        <string comment="LOT EDIT LABEL">LOT</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lotField">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string comment="LOT EDIT TOOLTIP">LOT</string>
         </property>
         <property name="maxLength">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="unitsLabel">
       <property name="toolTip">
        <string comment="PRODUCT COUNT EDIT TOOLTIP">PRODUCT COUNT</string>
       </property>
       <property name="text">
        <string comment="PRODUCT COUNT EDIT LABEL">PRODUCT COUNT</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="unitsField">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string comment="PRODUCT COUNT EDIT TOOLTIP">PRODUCT COUNT</string>
         </property>
         <property name="maxLength">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dispenseUnitLabel">
         <property name="toolTip">
          <string comment="DISPENSE UNITS DISPLAY TOOLTIP">DISPENSE UNITS</string>
         </property>
         <property name="text">
          <string notr="true">PH DISPENSE UNITS</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>expireField</tabstop>
  <tabstop>lotField</tabstop>
  <tabstop>unitsField</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
